docs/troubleshooting.md

```diff
@@ -26,14 +26,14 @@ wavefront   Healthy   Running (1/1)   Running (1/1)       Running (3/3)    Runni
 
 ### Proxy not running or unhealthy
 
-The proxy forwards logs, metrics, traces and spans from all other components to Tanzu Observability. No data flowing into Tanzu Observability likely means the proxy is failing.
+The Wavefront Proxy forwards logs, metrics, traces and spans from all other components to Operations for Applications. No data flowing into Operations for Applications likely means the proxy is failing.
```

Comment:
The Wavefront proxy forwards logs, metrics, traces, and spans from all other components to Operations for Applications. No data flowing into Operations for Applications means that the proxy might be failing.